jvcl/run/JvRichEdit.pas

```diff
@@ -6714,41 +6714,41 @@ function TJvTextAttributes.GetConsistentAttributes: TJvConsistentAttributes;
       AttrFlags[FType], LParam(@Format));
     with Format do
     begin
-      if (dwMask and CFM_BOLD) <> 0 then
+      if (dwEffects and CFM_BOLD) <> 0 then
```

Comment:
I disagree with this change, the documentation says this:

The dwMask member specifies which attributes are consistent throughout the entire selection

So clearly, a "GetConsistent" method should use the dwMask member.

See here:
https://docs.microsoft.com/en-us/windows/desktop/controls/em-getcharformat